ConsoleApp/ConsoleApp/Program.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+
+namespace ConsoleApp
+{
+    //i.safontev/homework/10-encapsulation
+   
+    enum Subjects
+    {
+        Math,
+        History,
+        Biology,
+        Chemistry,
+        Programming,
+        PhysicalCulrite,
+    }
+    enum DayOfWek
```

Comment:
just for your information, there is enum in .NET already
https://docs.microsoft.com/en-us/dotnet/api/system.dayofweek?view=net-6.0